components/nexus-launcher/src/main/java/org/sonatype/nexus/bundle/launcher/support/DefaultNexusBundle.java

```diff
@@ -246,6 +265,9 @@ protected void configure()
       sslPort = getConfiguration().getSslPort();
     }
 
+    this.secureUrl = new URL(getUrl().toExternalForm().replaceFirst("http://", "https://")
+      .replaceFirst(String.valueOf(getPort()), String.valueOf(getSslPort())));
```

Comment:
Would be safer to compose the SSL URL like we do with the app URL, just in case the hostname happens to contain the same numbers as the port. Otherwise "http://biz180.com:80/nexus" would become "http://biz1443.com:80/nexus".